Sources/MapboxNavigation/PreviewViewController.swift

```diff
@@ -4,12 +4,35 @@ import MapboxCoreNavigation
 import MapboxMaps
 import MapboxDirections
 
-// :nodoc:
+/**
+ `PreviewViewController` is a user interface for the preview mode experience. It allows to present
+ banners that contain information that is important for the user before actually starting turn-by-turn
+ navigation session (e.g. final destination presentation, main and alternative routes preview etc).
+ By default Mapbox Navigation SDK provides such banners:
+ - `DestinationPreviewViewController` - banner that is shown at the bottom of the screen and allows
+ to show information about the final destination, preview available routes and start active navigation
+ session
+ - `RoutesPreviewViewController` - banner that is shown at the bottom of the screen and allows to
+ preview information about the current `Route` (expected travel time, distance and expected time of arrival)
+ - `BannerDismissalViewController` - banner that is shown at the top of the screen and allows to
+ dismiss already presented banner
+ 
+ Internally `PreviewViewController` relies on two components:
+ - `NavigationMapView` - wraps `MapView` and provides convenience functions for adding and removing
+ route lines, route duration annotations, shows user location indicator etc
+ - `NavigationView` - wraps `NavigationMapView` and provides the ability to show drop-in related UI
```

Comment:
Isn't it too hard described?
**NavigationMapView** wraps MapView, but NavigationView wraps **NavigationMapView**, the same word NavigationMapView is used for different things

Reply:
I wanted to outline internal structure of the `PreviewViewController` as it'll hopefully allow users to better understand what APIs can be used. E.g. since `PreviewViewController` doesn't override `NavigationMapView`'s delegate users will have to set it themselves, this is is not the case for `NavigationViewController`.